### ISSUE TITLE
[Bug]: More info dialogue triggered from Browser_mod pop up showing behind the browser mod pop up

### ISSUE THREAD

**Issue by igiannakas** [piitaya/lovelace-mushroom#1211]
### Requirements

- [X] I checked the troubleshooting section in the README to verify that I have the latest Mushroom version.
- [X] I did a search to see if there is a similar issue or if a pull request is open.

### Current Behavior

With v3.03 when a more info dialogue triggered from an mushroom entity card residing in a browser_mod pop up, it is showing behind the browser mod pop up.

![Screenshot 2023-07-14 at 08 47 26](https://github.com/piitaya/lovelace-mushroom/assets/59056762/136a8c2b-79dc-4cf9-b88b-96f2b9cf899f)

![Screenshot 2023-07-14 at 08 47 46](https://github.com/piitaya/lovelace-mushroom/assets/59056762/5037fdf4-5840-49f7-a397-dfba59eec929)

In version 2.8.1 it works correctly.

![Screenshot 2023-07-14 at 08 54 09](https://github.com/piitaya/lovelace-mushroom/assets/59056762/fb642643-4928-4048-9291-151bd665c1a1)

For both, the latest revision of browser mod is used, which is fixing the dialogue not showing in the first place.

### Expected Behavior

The more info dialogue is shown infront of the browser mod pop up or the browser mod pop up is dismissed and the more info dialogue is showing.

### Steps To Reproduce

Version 3.03 mushroom cards, latest version of browser mod addressing defect 599 is installed
A browser mod pop up is created with a mushroom card in it
The mushroom card is linked to show the more info dialogue
Once clicked the more info dialogue is showing but behind the current pop up.

### Environment

```markdown
- Browser:Safari 16.5. Also on IOS 16.5.1
- HA Version: 2023.7.2
- Mushroom: 3.03
```

**Comment by piitaya** [comment]
I think it's a browser mod issue. You should have the same issue with the [button card](https://www.home-assistant.io/dashboards/button/)

**Comment by piitaya** [comment]
Fixes have been included in [2.3.0 browser mod release](https://github.com/thomasloven/hass-browser_mod/releases/tag/2.3.0)
